python/foglamp/services/core/server.py

```diff
@@ -590,13 +627,47 @@ def stop_storage(cls):
 
     @classmethod
     async def shutdown(cls, request):
-        pass
+        """ Shutdown the core microservice and its components
+
+        :return: JSON payload with message key
+        :Example:
+            curl -X POST http://localhost:<core mgt port>/foglamp/service/shutdown
+        """
+        try:
+            # There are 3 types of services
+            #   - Core
+            #   - Storage
+            #   - Southbound
+            try:
```

Comment:
I think we should not add this block, as these are started by scheduler hence scheduler should stop / shutdown these. 

Or
I should get all (not only southbound) and  
```
if svc._name == "FogLAMP Storage" or svc._name == "FogLAMP Core": 
    continue
else shutdown
```

Reply:
If this is shutdown all of FogLAMP, then it should

disable new tasks from running
 shut down the south microservices first
Then all tasks and south microservices are down then shutdown storage and finally exit

Reply:
I believe scheduler should stop all these tasks and plugins and here we should call only stop scheduler which we are already doing here